src/docs/arch-hs.dox

```diff
@@ -1,8 +1,272 @@
+    <td style="width: 92.4375px; height: 32px; text-align: center;">80</td>
+    <td style="width: 105.719px; height: 32px; text-align: center;">70</td>
+    <td style="width: 110.312px; height: 32px; text-align: center;">STANDARD</td>
+    <td style="width: 148.141px; height: 32px; text-align: center;">Value from U1</td>
+  </tr>
+  <tr style="height: 32px;">
+    <td style="width: 72.5156px; height: 32px; text-align: center;">1000</td>
+    <td style="width: 89.875px; height: 32px; text-align: center;">"AAA"</td>
+    <td style="width: 97.0312px; height: 32px; text-align: center;">80</td>
+    <td style="width: 93.9688px; height: 32px; text-align: center;">0</td>
+    <td style="width: 92.4375px; height: 32px; text-align: center;">90</td>
+    <td style="width: 105.719px; height: 32px; text-align: center;">80</td>
+    <td style="width: 110.312px; height: 32px; text-align: center;">STANDARD</td>
+    <td style="width: 148.141px; height: 32px; text-align: center;">Value from U2</td>
+  </tr>
+  <tr style="height: 32px;">
+    <td style="width: 72.5156px; height: 32px; text-align: center;">1000</td>
+    <td style="width: 89.875px; height: 32px; text-align: center;">"AAA"</td>
+    <td style="width: 97.0312px; height: 32px; text-align: center;">90</td>
+    <td style="width: 93.9688px; height: 32px; text-align: center;">0</td>
+    <td style="width: 92.4375px; height: 32px; text-align: center;">100</td>
+    <td style="width: 105.719px; height: 32px; text-align: center;">90</td>
+    <td style="width: 110.312px; height: 32px; text-align: center;">STANDARD</td>
+    <td style="width: 148.141px; height: 32px; text-align: center;">Value from U3</td>
+  </tr>
+  <tr style="height: 32px;">
+    <td style="width: 72.5156px; height: 32px; text-align: center;">
+      <strong>...</strong>
+    </td>
+    <td style="width: 89.875px; height: 32px; text-align: center;">
+      <strong>...</strong>
+    </td>
+    <td style="width: 97.0312px; height: 32px; text-align: center;">
+      <strong>...</strong>
+    </td>
+    <td style="width: 93.9688px; height: 32px; text-align: center;">
+      <strong>...</strong>
+    </td>
+    <td style="width: 92.4375px; height: 32px; text-align: center;">
+      <strong>...</strong>
+    </td>
+    <td style="width: 105.719px; height: 32px; text-align: center;">
+      <strong>...</strong>
+    </td>
+    <td style="width: 110.312px; height: 32px; text-align: center;">
+      <strong>...</strong>
+    </td>
+    <td style="width: 148.141px; height: 32px; text-align: center;">
+      <strong>...</strong>
+    </td>
+  </tr>
+</table>
+
+For the history store table entry corresponding to the update \c U1, the start timestamp in the key
+is the start timestamp for the update \c U1 and the stop timestamp in the value is the start
+timestamp for the update \c U2.
+
+When a modified history store table page is processed by reconciliation, certain records on the page
```

Comment:
`certain records on the page are removed when creating a new page image if the tombstones on those records have become globally visible`, could be reworded as:
`a new page image is created without records which have globally visible tombstones`, what do you think ?

Reply:
Thanks.